### ISSUE TITLE
There are some error.

### ISSUE THREAD

**Issue by wuzejinCN** [vagran/dxf-viewer#92]
1, Some arc's position is not correct.
2, Some rectangle was changed to tow triangles.
The testing file is attached. 
<img width="512" alt="Origin" src="https://github.com/vagran/dxf-viewer/assets/51487453/7ed7f32b-98cb-4605-abd3-a6d9361af796">
![output](https://github.com/vagran/dxf-viewer/assets/51487453/e920ddd0-5917-48c2-b279-10830c7f22e5)
[TestEntities.zip](https://github.com/vagran/dxf-viewer/files/13881717/TestEntities.zip)

**Comment by vagran** [comment]
Thanks for the nice test sample!

From the first look, it seems there are a lot of entities, which are arbitrary oriented in 3D space. The problematic arc probably has some arbitrary extrusion direction, which now is not fully supported because of 2D-only implementation. I will think about supporting an arbitrary projection plane. It is also an old problem with 3DFACE triangulation, I cannot get proper order of the vertices - an approach, which work on one file, breaks another file. I will revisit it once again.

![image](https://github.com/vagran/dxf-viewer/assets/6065976/1b328826-d6b1-45a6-9063-12e106ad7258)

Also checked in QCAD which has similar problems:
![image](https://github.com/vagran/dxf-viewer/assets/6065976/6cc6c81f-7c4f-461e-828e-ebba025544b7)

**Comment by wuzejinCN** [comment]
Indeed, due to the conversion from three dimensions to two dimensions for display, it would be better if this could be resolved. Thanks!
